README.md

```diff
@@ -1,131 +1,165 @@
 o-message [![Circle CI](https://circleci.com/gh/Financial-Times/o-message/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-message/tree/master)
 =================
 
-o-message is a messaging component used for alerting and informing. It can include variants on the type of message it delivers.
-
-- [Usage](#usage)
-	- [Markup](#markup)
-	- [JavaScript](#javascript)
-		- [Construction](#construction)
-		- [Options](#options)
-	- [Sass](#sass)
-- [Migration Guide](#migration-guide)
+`o-message` is a messaging component used for alerting, informing or making calls to action.
+
+- [Message Types](#message-types)
+	- [Action Message](#action-message)
+	- [Alert Message](#alert-message)
+	- [Notice Message](#notice-message)
+- [Markup](#markup)
+- [JavaScript](#javascript)
+	- [Construction](#construction)
+	- [Options](#options)
+- [Sass](#sass)
+- [Migration](#migration)
 - [Contact](#contact)
 - [Licence](#licence)
 
-## Usage
 `o-message` uses Sass and Javascript to show and hide a message component.
+
 It can be initialised declaratively if markup is provided on the page, or it can be initialised imperatively when using the [manual build process](http://origami.ft.com/docs/developer-guide/modules/building-modules/).
 
-`o-message` can initialise three types of messages: **alert**, **notice**, and **action**.
+### Message Types
+`o-message` provides three types of messages: **action**, **alert**, and **notice**.
+
+#### Action Message
```

Comment:
Action message is currently internal only. Maybe a table could flag support, o-table style https://github.com/Financial-Times/o-table#silent-mode?